src/main/java/org/sagebionetworks/web/client/widget/team/controller/TeamEditModalWidget.java

```diff
@@ -127,10 +143,67 @@ public void hide() {
 		view.hide();
 	}
 	
-	@Override
-	public void configure(Team team) {
+	public void configureAndShow(Team team) {
 		uploadedFileHandleId = null;
 		this.team = team;
+		teamId = Long.parseLong(team.getId());
+		//get the messaging parameter, and show
+		synapseClient.getTeamAcl(team.getId(), new AsyncCallback<AccessControlList>() {
+			public void onSuccess(AccessControlList result) {
+				teamACL = result;
+				show();
+			};
+			
+			@Override
+			public void onFailure(Throwable caught) {
+				synAlert.showError(caught.getMessage());
+			}
+		});
+	}
+	public void updateViewFromACL() {
+		//look for authenticated users principal ID from resource access set.
+		ResourceAccess ra = getResourceAccess(authenticatedUserGroupId);
+		if (ra != null) {
+			view.setAuthenticatedUsersCanSendMessageToTeam(ModelConstants.TEAM_MESSENGER_PERMISSIONS.equals(ra.getAccessType()));
+		} else {
+			view.setAuthenticatedUsersCanSendMessageToTeam(false);
+		}
+	}
+	
+	public void updateACLFromView() {
```

Comment:
please only add/remove the authenticated users principal ID, not the team principal ID